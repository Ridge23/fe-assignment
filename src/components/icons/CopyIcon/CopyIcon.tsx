import React from 'react';

import './sass/copy-icon.scss';

export function CopyIcon(): JSX.Element {
    return <i className="copy-icon" />
}