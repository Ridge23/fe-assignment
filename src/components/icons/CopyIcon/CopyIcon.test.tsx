import React from 'react';
import { shallow } from 'enzyme';

import CopyIcon from './CopyIcon'

describe('Footer', () => {
  it('should render correctly', () => {
    const component = shallow(<CopyIcon />);

    expect(component).toMatchSnapshot();
    expect(component.hasClass('copy-icon')).toBe(true);
  });
});