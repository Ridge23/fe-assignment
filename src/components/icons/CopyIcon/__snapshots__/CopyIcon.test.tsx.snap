// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CopyIcon should render correctly 1`] = `ShallowWrapper {}`;
